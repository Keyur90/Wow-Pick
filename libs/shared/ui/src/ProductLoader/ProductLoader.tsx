import { makeStyles } from '@material-ui/core/styles';
import Skeleton from '@material-ui/lab/Skeleton';
import React from 'react';

const useStyles = makeStyles((theme) => ({
  cardLoader: {
    marginBottom: theme.spacing(2),
    borderRadius: 0,
  },
  detailLoader: {
    borderRadius: 0,
  },
}));

const ProductLoader: React.FC = () => {
  const classes = useStyles();

  return (
    <>
      <Skeleton variant="rect" width={'100%'} height={166} className={classes.cardLoader} />
      <Skeleton variant="rect" width={'100%'} height={54} className={classes.detailLoader} />
    </>
  );
};

export { ProductLoader };
