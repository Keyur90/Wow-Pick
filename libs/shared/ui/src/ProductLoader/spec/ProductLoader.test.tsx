import React from 'react';
import { render } from '@testing-library/react';
import { ProductLoader } from '../ProductLoader';

test('renders ProductLoader', async () => {
  const { container } = render(<ProductLoader />);

  expect(container).toMatchSnapshot();
});
