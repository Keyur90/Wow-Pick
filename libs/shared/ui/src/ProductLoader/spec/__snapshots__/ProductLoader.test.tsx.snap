// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders ProductLoader 1`] = `
<div>
  <span
    class="MuiSkeleton-root MuiSkeleton-rect makeStyles-cardLoader-1 MuiSkeleton-pulse"
    style="width: 100%; height: 166px;"
  />
  <span
    class="MuiSkeleton-root MuiSkeleton-rect makeStyles-detailLoader-2 MuiSkeleton-pulse"
    style="width: 100%; height: 54px;"
  />
</div>
`;
